.github/workflows/test-publish-dry-run.yaml

```diff
@@ -0,0 +1,76 @@
+name: Test Publishing Dry Run
+
+on:
+  push:
+    branches:
+      - master
+      - feat/gradle-plugin-v2
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  publish-dry-run:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        agp: ["4.1.0"]
+        gradle: ["6.8.3"]
+        experimental: [false]
```

Comment:
what would be the use-case for this flag? I quite didn't understand.
if it's just because of `continue-on-error`, maybe we should use a different naming, like `continueOnError`

Reply:
This allows you to define a matrix entry like this:
```
      matrix:
        agp: ["7.0.0-alpha09"]
        gradle: ["7.0-20210309230237+0000"]
        experimental: [true]
```

That will run on the latest AGP/Gradle version but will run as optional (so you can monitor what's happening on the latest versions).
It would be useful to enable it when Gradle 7 enters beta/RC cycle.

Reply:
oh got it, so we allow experimental builds to fail and CI won't break, nice, thanks for explaining